repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,104 @@
 
 include::../_includes/in-progress.adoc[]
 
-<<add-header.adoc#,Add Header Filter>>::
-    Enables users to add new headers and header values to a request and/or response.
+== <<add-header.adoc#,Add Header Filter>>
```

Comment:
Seems odd to let the tool dictate what we can do, but I suppose it seems odd to want to include what are essentially (text of) links in the TOC requiring two clicks to get to what you want.  Why did we want them in the TOC again?  Or am I mis-remembering that and we didn't care about that?